app/helpers/equipment_models_helper.rb

```diff
@@ -7,4 +7,10 @@ def evaluate_img_presence(equipment_model)
       'no-image-260.gif'
     end
   end
+
+  def display_order(em)
+    category_orderings = EquipmentModel.where(category_id: em.category_id)
+                                       .map(&:ordering).sort
```

Comment:
this exact query appears at least twice (here and in your service object) -- I think it's worth making this a method on `EquipmentModel`